### PR TITLE
Yda 5393 copy to vault

### PR DESCRIPTION
**Pull request by claravox**

### CODE REVIEW

File: docs/design/processes/vault-process.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ submission.
 
 ## Copy to vault
 The **copy-accepted-folders-to-vault.r** cronjob in the tools directory of the research ruleset will copy any folder in 'ACCEPTED' state to the vault and set it to
```
Comment (claravox) [inline docs/design/processes/vault-process.md:59]: Will we still be using this cronjob?

File: docker/images/yoda_irods_icat/rules_uu.cfg [hunk 2]
```diff
@@ -79,6 +79,9 @@ arb_exempt_resources            = ''
 arb_min_gb_free                 = '0'
 arb_min_percent_free            = '0'
 
+copy_backoff_time               = '300'
```
Comment (claravox) [inline docker/images/yoda_irods_icat/rules_uu.cfg]: To discuss: what is a good default backoff time.

File: docs/design/processes/vault-process.md (hunk 1 above)
Comment (claravox) [reply docs/design/processes/vault-process.md:59]: Yes, we will.

File: roles/yoda_rulesets/defaults/main.yml [hunk 3]
```diff
@@ -40,6 +40,10 @@ async_revision_max_rss: 1000000000 # Limit the memory usage of a revision job
 enable_revision_cleanup: true      # Enable revision cleanup job (true/false)
 revision_cleanup_verbose_mode: false # Print extra information in revision cleanup job for troubleshooting (true/false)
 
+# Copy to vault configuration
+copy_backoff_time: 300           # How many seconds to wait before trying to copy a certain folder to vault again
```
Comment (lwesterhof) [inline roles/yoda_rulesets/defaults/main.yml]: Prefix with vault_

File: roles/yoda_rulesets/templates/rules_uu.cfg.j2 [hunk 4]
```diff
@@ -69,6 +69,9 @@ async_replication_max_rss      = '{{ async_replication_max_rss }}'
 async_revision_delay_time      = '{{ async_revision_delay_time }}'
 async_revision_max_rss         = '{{ async_revision_max_rss }}'
 
+copy_backoff_time              = '{{ copy_backoff_time }}'
```
Comment (lwesterhof) [inline roles/yoda_rulesets/templates/rules_uu.cfg.j2]: Prefix with vault_

**Review by lwesterhof — APPROVED** [review]
LGTM!

**Pull request merged**